docs/advanced_topics/proxy_models.rst

```diff
@@ -0,0 +1,49 @@
+.. _proxy_models:
+
+Using proxy models with Wagtail
+===============================
+
+For Django developers, `proxy models <https://docs.djangoproject.com/en/dev/topics/db/models/#proxy-models>`_
+can be a useful option when considering the design of models for a project. However, there are
+some areas in Wagtail where support for proxy models is limited. It is important to highlight that
+these limitations pertain specifically to the direct management of proxy models and their related
+permissions within Wagtail's admin UI - If you don't need to manage the data within Wagtail,
```

Comment:
At first read, "manage the data" feels a bit unspecific to me, especially for people new to modeladmin or Django. Could we be more specific here, something like

> It's perfectly fine to add references to proxy model instances, for example linking to one from a Wagtail Page. Where you'll have trouble is if you want to create, edit, or delete proxy model instances in the Wagtail UI using the ``modeladmin`` package. See below for more.

Reply:
Have tweaked your wording slightly to read:

> It's perfectly fine to add references to proxy model instances (for example, linking to one from a Wagtail Page). Where you're likely to have trouble is if you want to create, edit, or delete proxy model instances in the Wagtail UI using the `modeldamin` app, or by registering the model as a `Snippet`. See below for more information:

Reply:
Thanks, that looks great.